examples/job/expansions/job-expand.py

```diff
@@ -0,0 +1,46 @@
+#! /usr/bin/python
+# -*- coding: utf-8 -*-
+
+
+params = [
+		{
+			"name": "apple",
+			"args": r"./apple.txt",
+		},
+		{
+			"name": "banana",
+			"args": r"./香蕉.txt",
+		},
+		{
+			"name": "cherry",
+			"args": r"../вишня.doc",
+		}
+	]
+
+for p in params:
+  print (
```

Comment:
With this changes you can write down following usage:
```
mkdir ./jobs
python job-expand.py
kubectl create -f ./jobs
```
and user should be clear what to do from there on.